tests/tools/onert_train/README.md

```diff
@@ -34,3 +34,61 @@ onert_train \
 ```
 
 `onert_train --help` would help you to set each parameter.
+
+## Example
+
+To deliver a quick insight to use `onert_train`, let's train a simple mnist model. You could get a mnist tensroflow model code from [here](https://www.kaggle.com/code/amyjang/tensorflow-mnist-cnn-tutorial). 
+
+Before using `onert_train`, training data files and a model file have to be ready.
+
+### Prepare training data files
+
+`onert_train` expects that a preprocessed dataset is given as binary files. <br/>
+For convenience, we provide a tool([tf dataset convert](https://github.com/Samsung/ONE/tree/master/tools/generate_datafile/tf_dataset_converter)) that preprocesses tensorflow dataset and save it as binary files.
+
+You could use the tool like this. For detailed usage, please refer [here](https://github.com/Samsung/ONE/tree/master/tools/generate_datafile/tf_dataset_converter#readme).
+```bash
+# Move to tf_dataset_convert directory 
+$ cd ONE/tools/generate_datafile/tf_dataset_converter
+
+# install prerequisites
+$ pip3 install -r requirements.txt
+
+# generate binary data files
+$ python3 main.py \ 
+--dataset-name mnist \ 
+--prefix-name mnist \ 
+--model mnist 
+
+# check data files are generated
+# There are 'mnist.train.input.1000.bin' and 'mnist.train.output.1000.bin'
+$ tree out
+```
+
+### Prepare a circle model file
+
+`onert_train` use a `*.circle` file or a nnpackage as input. <br/>
+
+<!-- This readme is for the ONE developers, so they might know the onecc usage.--> 
```

Comment:
How about adding content to link to `onecc` usage?

Reply:
I'd love to. 
But is there a `onecc` usage document in public? I tried to find but I couldn't fine any document.